tests/mockserver.py

```diff
@@ -117,6 +117,14 @@ def render_POST(self, request):
                 "price": "10",
                 "currency": "USD",
             }
+            extract_from = request_data.get("productOptions", {}).get("extractFrom")
+            if extract_from:
+                from scrapy_zyte_api.providers import ExtractFrom
```

Comment:
Alternatively it can be moved to a different module (importing `providers` needs additional deps).